components/todoist/sources/common-task.js

```diff
@@ -0,0 +1,32 @@
+const todoist = require("../todoist.app.js");
+const common = require("./common.js");
+
+module.exports = {
```

Comment:
We should add a base implementation of `isElementRelevant` in this module just so that future event sources don't throw exceptions due to the fact that the method is missing (if a developer forgets to implement it):

I suggest this:
```js
module.exports = {
  methods: {
    isElementRelevant() {
      return true;
    },
  },
};
```